examples/source/interactivity-dynamic-content/ComboBox.html

```diff
@@ -101,27 +63,23 @@
           min-width: 320px;
           padding: var(--space-2);
         }
-        .combobox-container {
-          display: flex;
-          justify-content: center;
-          align-items: center;
-        }
-        .combobox-container label {
-          margin-right: var(--space-1);
-        }
     </style>
 </head>
 <body>
   <div class="preview-container">
-
       <!--
-        ## Data source
+        ## Autocomplete input
 
-        First we need to add a data source. Here we're using an inline `amp-state` declaration defining a list of `cities` in the US.
-        For long lists of suggestions, it's possible to fetch `amp-state` from a remote JSON endpoint using the `src` attribute.
+        First we need to add an `amp-autocomplete` element by giving it an input field and a data source. Here we're using an inline `script` declaration defining a list of cities in the US as `items`.
+        For long lists of suggestions, it's possible to fetch a remote JSON endpoint using the `src` attribute. 
+        
+        We will use `filter="token-prefix"` as our way to match on the user input and set `min-characters="0"` so that the combo box will display options even when the input is empty.
       -->
-      <amp-state id="cities">
-        <script type="application/json">[
+      <amp-autocomplete filter="token-prefix" min-characters="0" max-entries="5" on="select:AMP.setState({
```

Comment:
If you use `AMP.setState` you need to keep the amp-bind extension. Is the `city` variable used anywhere? If not you can remove it and keep amp-bind removed.

Reply:
Removed!